src/ui/screens.ts

```diff
@@ -0,0 +1,433 @@
+
+    if (lastState['isEmailMonitor']) {
+        readOnly = 'readOnly';
+        checked = 'checked';
+    }
+
+    if (lastState['isMailBoxMonitor']) {
+        disabledMailBoxes = 'disabled';
+        disabledFolders = 'disabled';
+        checkedTogle = 'checked';
+    }
+
+    let joplinFolders: JopinFolders;
+    const folders = [];
+
+    do {
+        joplinFolders = await joplin.data.get(['folders']);
+
+        for (let i = 0; i < joplinFolders.items.length; i++) {
+            const folder = joplinFolders.items[i];
+            let path = folder.title;
+            let node = folder;
+            const id = folder.id;
+
+            // The parent folder path
+            while (node.parent_id !== '') {
+                node = await joplin.data.get(['folders', node.parent_id]);
+                path = `${node.title}/${path}`;
+            }
+            folders.push({path: path, id: id});
+        }
+    } while (joplinFolders.has_more);
+
+    const options = [];
+
+    folders.forEach((folder) => {
+        const option = lastState['folderId'] === folder.id ? `<option selected value="${folder.id}">${folder.path}</option>` : `<option value="${folder.id}">${folder.path}</option>`;
+        options.push(option);
+    });
+
+
+    const mailBoxesOptions = [];
+    lastState['mailBoxes'].forEach((element) => {
+        const option = lastState['mailBox'] === element.path ? `<option selected value="${element.value}">${element.path}</option>`: `<option value="${element.value}">${element.path}</option>`;
+        mailBoxesOptions.push(option);
+    });
+
+
+    return `
+<div class="container">
+
+<div class="row" style="font-size: large;">
+
+  <div class="col-md-9 col-lg-7 col-xl-5 mx-auto">
+
+  <div class="card border-0 shadow rounded-3 my-3 align-middle d-flex justify-content-center" style="opacity: 0.97; top: 100px;" >
+
+      <div class="card-body p-4 p-sm-5">
+
+        <h1 class="card-title text-center mb-3 fw-light fs-1">Main Screen</h1>
+
+        <div class="container" style="text-align: center;">
+
+          <div class="input-group mb-3">
+            <span class="input-group-text" id="basic-addon1">From</span>
+            <input type="email" class="form-control" placeholder="email" aria-label="Email"
+              aria-describedby="basic-addon1" id='from' value = "${fromValue}" ${readOnly} required>
+          </div>
+          <div class="container" style="text-align: center">
+            <div class="form-check form-switch">
+              <input class="form-check-input" type="checkbox" role="switch" id="toggle-from"
+                onchange="toggle()" ${checked}>
+              <label class="form-check-label" for="toggle-from">Fetching & Monitoring</label>
+            </div>
+          </div>
+        </div>
+
+        <br>
+
+        <div class="container" style="text-align: center;">
+          <div class="input-group mb-3">  
+            <span class="input-group-text" id="basic-addon1">MailBoxes</span>
```

Comment:
Is MailBox the term used in the IMAP lib for folders?
Google has labels, Yahoo uses folders.
I think it's easy confuse mailbox with email account so maybe better call it folders?

Reply:
But I prefer "Mailbox" because it includes not only all folders or labels in Gmail, but also all primary mailboxes such as **inbox**, **spam**, **starred**, **important**, **all mail**, and so on. 

But if you prefer to rename the mailbox to the folder, no problem. I can change it now.

Reply:
> But if you prefer to rename the mailbox to the folder, no problem. I can change it now.

Maybe let's gather more feedback then.

Reply:
When I create the post about UI feedback in the forum, I will make a poll for this point.